File: Simulacao/MM2_Python/Uniforme.py
from FGVA import fgva

class uniforme(fgva):
    def __init__(self, a, b):
        fgva.__init__(self)
        self.a = a
        self.b = b
    def gerarNumero(self):
        return int(self.a + (self.b - self.a) * fgva.gerarNumero(self))

File: Simulacao/MM_1/distribuicao/uniforme.js
class Uniforme extends FVGA{

    constructor(limInf, limSup){
        super();
        this.limInf = limInf;
        this.limSup = limSup;
    }

    gerarNumero(){
        return parseInt(this.limInf + (this.limSup - this.limInf) * super.gerarNumero());
    }
}